config/config.exs

```diff
@@ -34,4 +34,10 @@ config :logger, :console,
   format: "$date $time $metadata[$level] $message\n",
   metadata: [:sdk, :method, :url]
 
+config :ext, :ext,
+  ws: [
+    subscriptions: [:subscription_1, :subscription_1],
+    endpoint: "ExtWeb.Endpoint can be in atom"
```

Comment:
`can be atom` ?

Reply:
yes

Reply:
I mean "ExtWeb.Endpoint can be in atom" -> "ExtWeb.Endpoint can be atom" :)


But if we use only Atom, let's add description: `ExtWeb.Endpoint should be atom`